_integration-schemas/adroll/v1/ads.md

```diff
@@ -0,0 +1,159 @@
+---
+tap: "adroll"
+version: "1"
+key: ""
+
+name: "ads"
+doc-link: "https://developers.nextroll.com/docs/crud-api/reference.html#get--api-v1-ad-get"
+singer-schema: "https://github.com/singer-io/tap-adroll/blob/master/tap_adroll/schemas/ads.json"
+description: |
+  The `{{ table.name }}` table contains information about the ads and creatives associated with the advertisables in your {{ integration.display_name }} account.
+
+replication-method: "Full Table"
+
+api-method:
```

Comment:
Similar issue to the endpoint in `ad_groups`, this isn't the endpoint the tap uses to get this data.